pkgs/development/python-modules/aspy.yaml/default.nix

```diff
@@ -0,0 +1,19 @@
+{ lib, buildPythonPackage, fetchPypi, pyyaml }:
+
+buildPythonPackage rec {
+  pname = "aspy.yaml";
+  version = "1.1.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1ajb97kn044ximkzq2090h4yblrhw77540pwiw345cp7mwzy4xqa";
+  };
+
+  propagatedBuildInputs = [ pyyaml ];
+
+  meta = with lib; {
+    description = "A few extensions to pyyaml.";
```

Comment:
No period: https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes (also for the other descriptions)